tools/deployment/osqueryctl

```diff
@@ -111,11 +111,11 @@ main() {
     EXAMPLE_CONFIG_PATH="/var/osquery/osquery.example.conf"
     PIDFILE="/var/osquery/osquery.pid"
     LOCKFILE="/var/osquery/osquery.lock"
-    EXEC="/usr/local/bin/osqueryd"
-    PLIST_DOMAIN="com.facebook.osqueryd"
+    EXEC="/opt/osquery.app/Contents/MacOS/osqueryd"
```

Comment:
Do we intent to have this land in `/opt/osquery.app` ? I was thinking something more like:
* `/opt/osquery/bin/osqueryd`  -- common to all platforms
* `/opt/osquery/???/osquery.app` -- macOS 

Whether `???` is `lib` `share` or nothing at all I don't know